### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BaptisteCecconi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +26,9 @@ Recommandation](http://www.ivoa.net/documents/latest/UCDlistMaintenance.html).
 ## Status?
 
 The last stable version is
-**[REC-1.3](http://www.ivoa.net/documents/UCD1+/20180527/index.html)**.
+**[EN-1.4](https://ivoa.net/documents/UCD1+/20210430/index.htmll)**.
```
Comment (molinaro-m) [inline README.md:29]: I think the date should reflect the endorsement, i.e. 20210616, before it was in Proposed EN status and the status change means also a date one (that can be reflected in the Makefile).

File: README.md [hunk 2]
```diff
@@ -25,9 +26,9 @@ Recommandation](http://www.ivoa.net/documents/latest/UCDlistMaintenance.html).
...
 
-Next version will be 1.4, as an _Endorsed Note_. It is prepared in this reporsitory. 
+Next version will be 1.5, as an _Endorsed Note_. It is prepared in this reporsitory.
```
Comment (molinaro-m) [inline README.md:31]: Followup of my previous comment, 1.5 should be here listed as Proposed Endorsed Note consequently

**Review by molinaro-m — COMMENTED** [review]
about the stds process status

File: README.md (hunk 2 above)
Comment (BaptisteCecconi) [reply README.md:31]: That is a good question. If we have an _Endorsed Note_, and we update it to a new minor version, it has to be proposed as a _Proposed Endorsed Note_ to TCG, and, if accepted, be switched to _Endorsed Note_ at this time. Is this the process?

File: README.md (hunk 1 above)
Comment (BaptisteCecconi) [reply README.md:29]: ok, let me close this PR (and the 2 other that are pending) and I'll do a complete PR from my fork with those changes.

File: README.md (hunk 2 above)
Comment (molinaro-m) [reply README.md:31]: That is (at least) my understanding.
It's like having a WD for version N.M+1 after you reach REC-N.M.

**Comment by msdemlei** [comment]
On Fri, Jun 18, 2021 at 12:12:34AM -0700, Baptiste Cecconi wrote:
> That is a good question. If we have an _Endorsed Note_, and we
> update it to a new minor version, it has to be proposed as a
> _Proposed Endorsed Note_ to TCG, and, if accepted, be switched to
> _Endorsed Note_ at this time. Is this the process? 

Pretty much.  Let my try to make it as unambigous as I can.

(1) Start the cycle: Set DOCVERSION to the *new* version, DOCTYPE to PEN (and of course DOCDATE)
(2) Have folks review things.
(3) After approval, set DOCTYPE to EN (and of course DOCDATE), follow the checklist for submitting from ivoatexDoc, upload.

And so on.

One thing that'll be exciting is when we get to version 1.9 in this way, which doesn't seem so far in the future.  You see, unfortunately our forefathers deemed that version 1.10 should be read as version 1.1.0, which is earlier than 1.9.  I've always wanted to get that changed but never had the nerve.  Anyone in for a conspiracy to get this fixed?

**Comment by molinaro-m** [comment]
I thought the "new" versioning for docs was N.M with N,M in [0,...[ without micro versioning.
Thus 1.10 should be 1.9 with another minor revision.
Things like 1.03, 1.23 should be part of the "old" heritage.
I'll re-check the DocStdProc to see if it's clear there or not.

**Comment by molinaro-m** [comment]
The text says:

`The number to the right of the decimal point is an integer counter, beginning with 0
and increasing in simple cardinal order (0, 1, 2, ... 9, 10, 11, ...). This number does not
track every revision to a document, but rather, denotes a logical version or conceptually
consistent view. This number should only be incremented when there are significant
and substantial changes to text but few (minor) or no changes required of
implementations. The version number normally remains fixed as a document is
promoted from Working Draft to Proposed Recommendation to Recommendation, with
editorial revisions indicated by the change of date.`

I read it as 1.10 comes after 1.9. Issue _could_ still be version 1.10 (and 1.11, etc..), even if 
it will be EN-1.10 and not REC-1.10 (already existing).

**Comment by BaptisteCecconi** [comment]
(the original sin was thus to keep version 1 to UCDs when UCD 1+ were issued... it should have been UCD 2.0...)

**Comment by msdemlei** [comment]
On Fri, Jun 18, 2021 at 01:51:45AM -0700, Marco Molinaro wrote:
> The text [of standards and processes] says:
> 
> The number to the right of the decimal point is an integer counter, beginning with 0
> and increasing in simple cardinal order (0, 1, 2, ... 9, 10, 11, ...). This number does not
> track every revision to a document, but rather, denotes a logical version or conceptually

Oh wow, that's nice.  It seems it's been like this since REC-1.2 of
DocStd.  DocStd 1.0 appears to be quiet on this...  Hm.

This has been different in the past, and we still have quite a few
old-style versions around ("SCS 1.03").  I think this was done
according to
https://ivoa.net/documents/Notes/DocStd/Procedures-20040425.html#VersionNumbers
(which is still in "IVOA Procedures", with a version number of
"1.00"...), but I'm not sure where else this "compare version numbers
as floats" idea might also be around.

Good to know that DocStd has done it right; I was worried in
particular for the UDF note...

**Comment by BaptisteCecconi** [comment]
Closing this PR. See #34

**Pull request closed without merging**